src/Features/Core/Portable/InlineHints/IInlineHintsService.cs

```diff
@@ -6,14 +6,13 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.LanguageService;
-using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.InlineHints
 {
     internal interface IInlineHintsService : ILanguageService
     {
-        Task<ImmutableArray<InlineHint>> GetInlineHintsAsync(Document document, TextSpan textSpan, InlineHintsOptions options, CancellationToken cancellationToken);
+        Task<ImmutableArray<InlineHint>> GetInlineHintsAsync(
+            Document document, TextSpan textSpan, InlineHintsOptions options, bool displayAllOverride, CancellationToken cancellationToken);
```

Comment:
@tmat i could also make this part of `InlineHintsOptions`.  It just wouldn't be populated from any options-service, but would be provided directly by the feature. However, i wasn't certain if that was a pattern we wanted for our immutable-options/options-services in the future.  So, for now, i pulled out 'displayAllOverride' as a separate explicit option that is passed along.

If it is fine to move into InlineHintsOptions, i'm happy to move into it.  BUt i wanted to check with you.